CalibCalorimetry/CaloTPG/src/CaloTPGTranscoderULUT.cc

```diff
@@ -204,12 +204,29 @@ CaloTPGTranscoderULUT::getOutputLUTSize(const HcalTrigTowerDetId& id) const
       case HcalTopologyMode::TriggerMode_2016:
          return QIE8_OUTPUT_LUT_SIZE;
       case HcalTopologyMode::TriggerMode_2017:
+         if (id.ietaAbs() <= theTopology->lastHERing())
+            return QIE8_OUTPUT_LUT_SIZE;
+         else
+            return QIE10_OUTPUT_LUT_SIZE;
+      case HcalTopologyMode::TriggerMode_2017plan1:
+         if (theTopology->dddConstants()->isPlan1(id))
+            return QIE11_OUTPUT_LUT_SIZE;
```

Comment:
hello @matz-e 
detIdSp_ https://github.com/matz-e/cmssw/blob/9c7c69d4203cf4d932921a4d4103956b8dc7cfbd/Geometry/HcalCommonData/interface/HcalDDDRecConstants.h#L136   is empty by construction, for now
which in turns means isPlan1(...) always returns false.